# HG changeset patch
# User Arlo Breault <arlolra@gmail.com>
# Date 1464281776 25200
#      Thu May 26 09:56:16 2016 -0700
# Node ID f7624aac9022c38cfec6b2bc4f73ad69a5cf2a04
# Parent  664eff8ea96f4bb2c15aa3ff01fd7c33e5cc3480
Notification should use message prepared for dislaying

diff --git a/im/modules/ibNotifications.jsm b/im/modules/ibNotifications.jsm
--- a/im/modules/ibNotifications.jsm
+++ b/im/modules/ibNotifications.jsm
@@ -20,17 +20,17 @@ var Notifications = {
     } catch (e) { }
     return ellipsis;
   },
 
   _showMessageNotification: function (aMessage) {
     // Put the message content into a div node of the hidden HTML window.
     let doc = getHiddenHTMLWindow().document;
     let xhtmlElement = doc.createElementNS("http://www.w3.org/1999/xhtml", "div");
-    xhtmlElement.innerHTML = aMessage.message.replace(/<br>/gi, "<br/>");
+    xhtmlElement.innerHTML = aMessage.displayMessage.replace(/<br>/gi, "<br/>");
 
     // Convert the div node content to plain text.
     let encoder =
       Components.classes["@mozilla.org/layout/documentEncoder;1?type=text/plain"]
                 .createInstance(Components.interfaces.nsIDocumentEncoder);
     encoder.init(doc, "text/plain", 0);
     encoder.setNode(xhtmlElement);
     let messageText = encoder.encodeToString().replace(/\s+/g, " ");
